testdata/htsjdk/variant/HiSeq.10000.vcf

```diff
@@ -10,6 +10,8 @@
 ##FILTER=<ID=Indel,Description="Overlaps a user-input mask">
 ##FILTER=<ID=LowQual,Description="Low quality">
 ##FILTER=<ID=LowQual,Description="QUAL  50.0">
+##FILTER=<ID=ANNOTATION,Description="ANNOTATION != \"NA\" || ANNOTATION <= 0.01">
+##FILTER=<ID=ANNOTATION2,Description="ANNOTATION with quote \" that is unmatched">
```

Comment:
Is this file used by tests for the legacy VCF3 codec? (since I think you only changed the VCF4 parser)